package.json

```diff
@@ -24,8 +24,8 @@
     "@types/node": "^13.7.7",
     "@vtex/prettier-config": "^0.1.3",
     "eslint": "^6.8.0",
-    "eslint-config-vtex": "^12.2.1",
-    "eslint-config-vtex-react": "^6.2.1",
+    "eslint-config-vtex": "^12.0.3",
```

Comment:
need to keep consistency between the controllers. In the near future, we can update it again.